test/common/stats/thread_local_store_test.cc

```diff
@@ -37,7 +38,7 @@ class StatsThreadLocalStoreTest : public testing::Test, public RawStatDataAlloca
     }));
 
     EXPECT_CALL(*this, alloc("stats.overflow"));
-    store_.reset(new ThreadLocalStoreImpl(*this));
+    store_.reset(new ThreadLocalStoreImpl(options_, *this));
```

Comment:
Sorry, tossing in a new comment because replies don't seem to show up at the bottom.  nit: can we use std::make_unique here?

Reply:
Done.